src/main/resources/application.yml

```diff
@@ -2,11 +2,11 @@ server:
   port: 8081
   servlet.context-path: /sandbox
 datasource:
-  schema: 'sandbox'
-  host: '${dockerdbhost:localhost}:3306'
+  schema: 'workplace'
+  host: '${dockerdbhos:localhost}:3306'
```

Comment:
Please set back the configuration

Reply:
### You can just merge master to this config files by not accepting my configs.